src/common/obj_bencher.cc

```diff
@@ -228,6 +228,7 @@ int ObjBencher::aio_bench(
     return -EINVAL;
 
   int num_objects = 0;
+  int num_writes = 0;
```

Comment:
@krehm (NIT) Please consider to keep PR title < 50 characters for future PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits